lib/compilers/snowball.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2016, Compiler Explorer Authors
```

Comment:
Can you update this copyright year please?

Reply:
sure